packages/cron/src/jobs/names.js

```diff
@@ -0,0 +1,73 @@
+import debug from 'debug'
+import retry from 'p-retry'
+
+const MAX_REQUEST_PAGE = 10
+const W3NAME_API_URL_STAGING = 'https://name.web3.storage/'
+const W3NAME_API_URL_PRODUCTION = 'https://name-staging.web3.storage/'
```

Comment:
@adamalton @francois-potato Is this reversed for a reason? Should staging point to staging instead?

Reply:
Ooh, no! Good spot!